src/authenticator_delegate.rs

```diff
@@ -94,10 +94,11 @@ async fn present_user_code(device_auth_resp: &DeviceAuthResponse) {
         device_auth_resp.user_code, device_auth_resp.verification_uri
     );
     println!("Do not close this application until you either denied or granted access.");
-    println!(
-        "You have time until {}.",
-        device_auth_resp.expires_at.with_timezone(&Local)
-    );
+    let printable_time = match UtcOffset::current_local_offset() {
+        Ok(offset) => device_auth_resp.expires_at.to_offset(offset),
+        Err(_) => device_auth_resp.expires_at,  // Fallback to printing in UTC
+    };
```

Comment:
I don't know if this works as you want it to and I don't think you have a test for it. You may want to verify.

Reply:
It doesn't seem to be able to determine my timezone and falls back to UTC; the previous version handled this correctly. It isn't a blocker for me personally, as I believe this feature to only be used rarely (and the displayed fact is still correct). Nevertheless, if you know a typical reason for `time` to not be able to find the timezone (I also tested just displaying the local time, which also didn't work) off the top of your head, this would be the cherry on top.

Reply:
I'm afraid I don't know why it wouldn't be working for you. Sorry :( The implementation does try to cover all platforms - https://github.com/time-rs/time/tree/main/src/sys/local_offset_at

Reply:
That's perfectly fine, thank you for the whole effort in any case! :)